src/ai/ai.hpp

```diff
@@ -137,8 +137,9 @@ class AI {
 
 	/**
 	 * Get the number of days before the next AI should start.
+	 * @param count For which Nth AI to retrieve the starting date.
```

Comment:
I still don't know what to do here. Feels like something is missing. GetStartNextTime with a parameter has a very specific use for that loop in MaybeStartNewCompany. It is called in the middle of it. For each ai that is to start, ais_to_start count increases. When checking for the next AI to start, it has to skip those that were already accounted for.

"For which Nth AI to retrieve the starting date." - I don't know, because when looking at the function, it is only looking for non valid companies where an AI could possibly start, and retrieves the start_date of that position. But the position is not necessarily ordered. "Nth" feels wrong.

Reply:
Nevermind